apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+export default class SpriteUpload extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      file: null
+    };
+  }
+
+  handleSubmit(event) {}
+
+  handleChange = event => {
+    this.setState({
+      file: URL.createObjectURL(event.target.files[0])
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>Sprite Upload</h1>
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            <h4> Sprite Category: </h4>
+            <input type="text" />
+          </label>
+          <label>
+            <h4> Select Sprite to Add to Library: </h4>
+            <input type="file" ref="uploader" onChange={this.handleChange} />
```

Comment:
Is there a way to ensure the selected file is a png?

Reply:
Yes! I'll add that in during the next step (uploading to S3)

Reply:
(Added it to the next PR in this task)

Reply:
(Presumably using the `accept` [attribute](https://stackoverflow.com/questions/3828554/how-to-allow-input-type-file-to-accept-only-image-files)?)

Reply:
(Yup! `input type="file"` is magic)